File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/Java/Max.java
import java.io.*;


class Max{

	public static void main(String args[])throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		System.out.printf("Enter the number1 : ");
		int num1 = Integer.parseInt(br.readLine());
		System.out.printf("Enter the number2 : ");
		int num2 = Integer.parseInt(br.readLine());
		System.out.printf("Enter the number3 : ");
		int num3 = Integer.parseInt(br.readLine());

		if(num1>num2 && num1>num3)
			System.out.println("The Maximum number amongst"+num1+" "+num2+" & "+num3+" is: "+num1);
		else if(num2>num1 && num2>num3)
			System.out.println("The Maximum number amongst"+num1+" "+num2+" & "+num3+" is: "+num2);
		else
			System.out.println("The Maximum number amongst"+num1+" "+num2+" & "+num3+" is: "+num3);

	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/Python/Max.py
num1 = input("Enter number1 : ")
num2 = input("Enter number2 : ")
num3 = input("Enter number3 : ")

if num1>num2 and num1>num3:
    print(str(num1)+" is Maximum")
elif num2>num1 and num2>num3:    
    print(str(num2)+" is Maximum")
elif num3>num1 and num3>num2:
    print(str(num3)+" is Maximum")

